packages/dds/merge-tree/src/test/partialLength.spec.ts

```diff
@@ -33,6 +33,7 @@ describe("partial lengths", () => {
                 segment.isLeaf()
                 && !(segment.removedSeq !== undefined && segment.removedSeq >= seq)
                 && segment.localRemovedSeq === undefined
+                && (segment.seq === undefined || segment.seq <= seq)
```

Comment:
this isn't quite right for when the segment is locally inserted but un-acked (there, `localSeq` will be some positive number and `segment.seq` will be `UnassignedSequenceNumber`, which is -1).